src/components/znkTimeline/module.js

```diff
@@ -0,0 +1,18 @@
+(function (angular) {
+    'use strict';
+    var svgMap = {
+        drill: 'components/znkTimeline/svg/icons/timeline-drills-icon.svg' ,
+        game: 'components/znkTimeline/svg/icons/timeline-mini-challenge-icon.svg' ,
+        tutorial: 'components/znkTimeline/svg/icons/timeline-tips-tricks-icon.svg' ,
+        section: 'components/znkTimeline/svg/icons/timeline-diagnostic-test-icon.svg',
+        practice: 'components/znkTimeline/svg/icons/timeline-test-icon.svg'
+    };
+    angular.module('znk.infra.znkTimeline', ['znk.infra.svgIcon', 'znk.infra.enum'])
+        .config([
+            'SvgIconSrvProvider',
+            function (SvgIconSrvProvider) {
+                SvgIconSrvProvider.registerSvgSources(svgMap);
+            }])
+        .constant('timelineImages', svgMap);
```

Comment:
I dont think it should be a constant since no one is using it besides the time line service ....